### ISSUE TITLE
Crystal... in menu bar not consistent with other headings

### ISSUE THREAD

**Issue by matterhorn103** [OpenChemistry/avogadrolibs#1528]
For consistency, the `Crystal...` heading should not be written with an ellipsis in the menu bar.

None of the other headings have it, and in general only menu options that open dialog windows have them.

**Comment by matterhorn103** [comment]
I can't find where in the code the menu options are set up sadly, despite doing a global search for "Crystal…"

**Comment by ghutchis** [comment]
It shouldn't be. What localization are you using?

https://hosted.weblate.org/search/avogadro/-/en_GB/?q=Crystal&sort_by=-priority%2Cposition&checksum=

**Comment by matterhorn103** [comment]
I could have sworn I checked that I was on the US English version before opening the issue. Apparently not, so apologies.

This only happens in the version I have compiled though, not the AppImage. I suspect it might be one of the translations I fixed in the en_gb localization a month or two ago, and I simply have an outdated version of the localizations in my source directory.

Until now I've only been rebasing `avogadroapp` and `avogadrolibs`. Hadn't clicked that `avogadro-i18n` is the translations and also needs rebasing. Apologies!
